project/AkkaBuild.scala

```diff
@@ -332,6 +339,9 @@ object AkkaBuild extends Build {
     licenses := Seq(("Apache License, Version 2.0", url("http://www.apache.org/licenses/LICENSE-2.0"))),
     homepage := Some(url("http://akka.io/")),
 
+    // don't publish test artifacts
+    publishSigned in Test := (),
```

Comment:
It is strange that this is needed since we do not do `test:publishSigned`. Is `AkkaBuild.dontPublishSettings` alone not enough?